src/Lib/Canvas.vala

```diff
@@ -673,31 +771,24 @@ public class Akira.Lib.Canvas : Goo.Canvas {
                 set_cursor_by_edit_mode ();
                 break;
             case Nob.TOP_LEFT:
-                set_cursor (Gdk.CursorType.TOP_LEFT_CORNER);
-                break;
-            case Nob.TOP_CENTER:
-                set_cursor (Gdk.CursorType.TOP_SIDE);
-                break;
-            case Nob.TOP_RIGHT:
-                set_cursor (Gdk.CursorType.TOP_RIGHT_CORNER);
-                break;
-            case Nob.RIGHT_CENTER:
-                set_cursor (Gdk.CursorType.RIGHT_SIDE);
-                break;
             case Nob.BOTTOM_RIGHT:
-                set_cursor (Gdk.CursorType.BOTTOM_RIGHT_CORNER);
```

Comment:
Any particular reason why you switched to name based cursors?
I have nothing against it, I'm just curious.